modules/react/loading-dots/stories/examples/Accessible.tsx

```diff
@@ -1,6 +1,39 @@
 import React from 'react';
 import {LoadingDots} from '@workday/canvas-kit-react/loading-dots';
+import {base, system} from '@workday/canvas-tokens-web';
+import {Flex} from '@workday/canvas-kit-react/layout';
+import {SecondaryButton} from '@workday/canvas-kit-react/button';
+import {createStyles, cssVar} from '@workday/canvas-kit-styling';
+import {AccessibleHide, AriaLiveRegion} from '@workday/canvas-kit-react/common';
 
 export const Accessible = () => {
-  return <LoadingDots aria-live="polite" aria-label="Loading Users" />;
+  const [loadingState, setLoadingState] = React.useState('idle');
+  const loadingStyles = createStyles({
+    backgroundColor: base.licorice300,
+    padding: system.space.x3,
+  });
+
+  React.useEffect(() => {
+    if (loadingState === 'loading') {
+      setTimeout(() => {
+        setLoadingState('success');
+      }, 4000);
+    }
```

Comment:
Anytime there's a `setTimeout` (or any other async code that needs a cleanup), there should be a `clearTimeout`. Without this, there will be React warnings about trying to set state on unmounted components if the story is unrendered during the 4 seconds it takes to call the `setLoadingState`. The simplest way to do this without a lot of TypeScript code is to move the `if` statement into the timer callback:

```suggestion
    const timer = setTimeout(() => {
      if (loadingState === 'loading') {
        setLoadingState('success');
      }
    }, 4000);

    return () => {
      clearTimeout(timer);
    };
```